roles/ceph-validate/tasks/check_rgw_multisite.yml

```diff
@@ -55,9 +55,9 @@
     - rgw_zonesecondary
     - rgw_pull_proto is undefined
 
-- name: fail if rgw_pullhost is not set
+- name: fail if rgw_pull_host is not set
   fail:
-    msg: "rgw_pullhost has not been set by the user"
+    msg: "rgw_pull_host has not been set by the user"
   when:
     - rgw_zonesecondary
-    - rgw_pullhost is undefined
+    - rgw_pull_host is undefined
```

Comment:
not sure about this condition given that you have a default value in `roles/ceph-defaults/defaults/main.yml`